tests/Aspire.Hosting.Tests/ManifestGenerationTests.cs

```diff
@@ -328,37 +328,6 @@ public void EnsureAllPostgresManifestTypesHaveVersion0Suffix()
         Assert.Equal("value.v0", db.GetProperty("type").GetString());
     }
 
-    [Fact]
-    public void EnsureAllAzureStorageManifestTypesHaveVersion0Suffix()
-    {
-        var program = CreateTestProgramJsonDocumentManifestPublisher();
-
-        var parent = program.AppBuilder.AddAzureStorage("storage");
-        parent.AddBlobs("blobs");
-        parent.AddQueues("queues");
-        parent.AddTables("tables");
-
-        // Build AppHost so that publisher can be resolved.
-        program.Build();
-        var publisher = program.GetManifestPublisher();
-
-        program.Run();
-
-        var resources = publisher.ManifestDocument.RootElement.GetProperty("resources");
-
-        var storage = resources.GetProperty("storage");
-        Assert.Equal("azure.storage.v0", storage.GetProperty("type").GetString());
-
-        var blobs = resources.GetProperty("blobs");
-        Assert.Equal("azure.storage.blob.v0", blobs.GetProperty("type").GetString());
```

Comment:
These tests will probably go away eventually ... but for now just change them to value.v0 for the sub-resources?

Reply:
I have replacement tests for this alraedy.